pyirf/benchmarks/angular_resolution.py

```diff
@@ -34,14 +34,16 @@ def angular_resolution(
         Table containing the 68% containment of the angular
         distance distribution per each reconstructed energy bin.
     """
-
     # create a table to make use of groupby operations
     table = Table(events[[f"{energy_type}_energy", "theta"]])
 
     table["bin_index"] = calculate_bin_indices(
         table[f"{energy_type}_energy"].quantity, energy_bins
     )
 
+    n_bins =  len(energy_bins) - 1
```

Comment:
we did this in many places: it is quite confusing when reading it - `energy_bins` should be really called `energy_bins_edges`

Reply:
We took it from numpy conventions.